### ISSUE TITLE
Unable to enter comma

### ISSUE THREAD

**Issue by pztrn** [zuoez02/siyuan-plugin-whiteboard#21]
When entering some text e.g. in square I unable to enter comma (`,`). It activates some sort of accessibility thing and actually works as mouse button. Comman can be successfully copy-pasted into element's text.

It works on tldraw.com and looks like not related to SiYuan itself as commas can be entered in notes.

Plugin version v0.1.10
